vault/logical_system_paths.go

```diff
@@ -1479,8 +1760,16 @@ func (b *SystemBackend) monitorPath() *framework.Path {
 				Default:     "standard",
 			},
 		},
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.ReadOperation: b.handleMonitor,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation: &framework.PathOperation{
+				Callback: b.handleMonitor,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields:      map[string]*framework.FieldSchema{},
+					}},
+				},
```

Comment:
looks like this is no content
https://github.com/hashicorp/vault/blob/901876f27abc9e74904aaebaba7a31bc9cb970a6/vault/logical_system.go#L3771